lib/request-wrapper.ts

```diff
@@ -367,7 +449,8 @@ export class RequestWrapper {
       instance: axiosInstance,
       backoffType: 'exponential',
       checkRetryAfter: true, // use Retry-After header first
-      maxRetryDelay: 30 * 1000, // default to 30 sec max delay
+      maxRetryDelay: 30 * 1000, // default to 30 sec max delay,
```

Comment:
```suggestion
      maxRetryDelay: 30 * 1000, // default to 30 sec max delay
```
Feel free to ignore this, but I don't think that extra comma at the end of the comment is needed. :)

Reply:
fixed